AUTHORS

```diff
@@ -31,6 +31,7 @@ Daniel Nichter <nil at codenode.com>
 Daniël van Eeden <git at myname.nl>
```

Comment:
Is adding your name to AUTHORS part of the contributing guidelines? I wouldn't consider Dockerfiles/Mayhemfiles core authorship.

Reply:
It appears to be, to quote the contributing guidelines:

> By contributing to this project, you share your code under the Mozilla Public License 2, as specified in the LICENSE file. Don't forget to add yourself to the AUTHORS file.

It's also part of the checklist to submit a commit. While I do agree that adding Dockerfiles/mayhemfiles to an already harnessed repository is not core authorship, I was trying to follow the guidelines as they were written.

What are your thoughts?